Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,26 @@
+require "English"
```

Comment:
Remove this.

Reply:
@marivera Try taking a look at https://www.rubydoc.info/github/Homebrew/brew/master/Homebrew/Assertions